glean-core/src/internal_pings.rs

```diff
@@ -19,9 +19,9 @@ pub struct InternalPings {
 }
 
 impl InternalPings {
-    pub fn new() -> InternalPings {
+    pub fn new(enabled: bool) -> InternalPings {
         InternalPings {
-            baseline: PingType::new(
+            baseline: PingType::new_internal(
```

Comment:
Add another layer to the 🧅!

Reply:
yeah, I don't like it really, but due to how we handle those pings with an `Arc` internally it's hard to modify their `enabled` value otherwise.